ql/src/test/queries/clientpositive/pcs.q

```diff
@@ -58,7 +58,7 @@ SORT BY A.key, A.value, A.ds;
 explain extended select ds from pcs_t1 where struct(case when ds='2000-04-08' then 10 else 20 end) in (struct(10),struct(11));
 select ds from pcs_t1 where struct(case when ds='2000-04-08' then 10 else 20 end) in (struct(10),struct(11));
 
-explain extended select ds from pcs_t1 where struct(ds, key, rand(100)) in (struct('2000-04-08',1,0.2), struct('2000-04-09',2,0.3));
+explain extended select ds from pcs_t1 where struct(ds, key) in (struct('2000-04-08',1), struct('2000-04-09',2));
```

Comment:
If we remove the RAND we are losing in terms of test coverage which was actually useful to detect this regression. We should strive to increase coverage not the other way around.

Reply:
Once we are ready to merge this PR we should create a JIRA ticket to document the regression and potentially handle it in a follow-up.